internal/controller/authenticator.go

```diff
@@ -162,6 +164,14 @@ func SetUpOAuthConnection(
 	}
 	tokenSource := GetTokenSource(contextWithTimeout, oauthConfig, tlsConfig)
 
+	// Log proxy settings
+	logger.Info(
```

Comment:
Remove this?? It's redundant with the logging of `HTTPS_PROXY` already done in `main.go`:
https://github.com/illumio/cloud-operator/pull/195/files#diff-c444f711e9191b53952edb65bfd8c644419fc7695c62611dc0fb304b4fb197d6R154